packages/chat-component-bindings/review/chat-component-bindings.api.md

```diff
@@ -12,7 +12,7 @@ import { ChatErrors } from '@internal/chat-stateful-client';
 import { ChatMessageWithStatus } from '@internal/chat-stateful-client';
 import { ChatParticipant } from '@azure/communication-chat';
 import { ChatThreadClient } from '@azure/communication-chat';
-import { CommonProperties } from '@internal/acs-ui-common';
+import { Common } from '@internal/acs-ui-common';
```

Comment:
Why is there an API change without any corresponding code changes?

Did this leak in from the local environment?

Reply:
just did a rush update and build ...